spec.md

```diff
@@ -325,16 +343,24 @@ on the definition of OPTIONAL.
   event format defines the relationship in
   [section 3.1](./json-format.md#31-handling-of-data).
 
-  When this attribute is omitted, `data` simply follows the event
-  format's encoding rules. For the JSON event format, the `data` value
-  can therefore be a JSON object, array, or value.
+  Some event formats have a default value for `datacontenttype`. Events in such
```

Comment:
It feels odd to talk about a "default content type" because it comes across like something specific to the format - almost like you're saying JSON defines a default content type and that's not really what's going on. All we're doing is saying that `datacontenttype` can be omitted if it's the same value as the event format itself - just an optimization. Can we just say that instead?

Reply:
It's not the same. The media type for a JSON-formatted event is "application/cloudevents+json". The media  type for an arbitrary JSON value is "application/json". It is obvious, but it needs to be explicit *as a media type* for use in non-JSON contexts. Other formats/bindings may not have a less obvious default, or none at all. 

This thread started with confusion about the AMQP type system not being expressible in a cloud event. I'm trying to say that is because AMQP doesn't *have* a default media type, therefore any CE data ever carried over AMQP must be described by some other media type. Formats that do have a default media type (JSON) will get tagged with it as they enter the AMQP world and can be properly parsed when they exit.

Reply:
I agree with the intent of this. I would prefer 'implied' over 'default':

Some event formats have an implied value for datacontenttype. For example, in the JSON format `datacontenttype` has an implicit value of "application/json", which permits for `data` to be any JSON value. If overriden with a different than the implied value, `data` must follow the respectuve content-type's rules.

Reply:
Reworded, back to "implied".